playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -586,10 +586,22 @@ EDXAPP_CODE_JAIL_LIMITS:
 
 # Set the number of workers explicitly for lms and cms
 # Should be set to
-# EDXAPP_WORKERS:
-#   lms: <num workers>
-#   cms: <num workers>
+EDXAPP_WORKERS:
+  lms: 14
+  cms: 14
+# EDXAPP_WORKER_THREADS
+#   lms: <num worker threads>
+#   cms: <num worker threads>
 EDXAPP_WORKERS: !!null
+EDXAPP_WORKER_THREADS: !!null
+EDXAPP_WORKER_CLASS: !!null
+# Gunicorn will restart each worker after x amount of
+# requests processed. The x value will be randomly selected between
+# 1 and this value (default 1500)
+EDXAPP_CMS_MAX_REQUEST_JITTER: 1500
+EDXAPP_LMS_MAX_REQUEST_JITTER: 1500
```

Comment:
Will this override `EDXAPP_LMS_MAX_REQ`?

Is there some reason we don't want a specific number of requests, but instead a random number?